### PR TITLE
Bouncing ball-Animation

### PR DESCRIPTION
**Pull request by Anurag-singh-thakur**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [ ] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x ] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [ x] Your pull request contains `index.html` and `styles.css`.
	- [x ] Your html file is named exactly: `index.html`
	- [x ] Your css file is named exactly: `styles.css`
- [ ] Your pull request your `meta.json` with:
	- [ ] `githubHandle`: being your unique github user name,
	- [ ] `artName` : name of your animation
- [ x] Your submission includes **at least one animation**.
- [x ] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/Anurag-singh-thakur-Bouncing-Ball-Animation/index.html
Comment (LaurelineP) [inline Art/Anurag-singh-thakur-Bouncing-Ball-Animation/index.html:1]: # Incorrect link href source
- [ ] Once you've renamed your file as the PR checklist recommended, 
please adjust accordingly your href with the correct style name

File: Art/Anurag-singh-thakur-Rotating Sqaure/.vscode/settings.json
Comment (LaurelineP) [inline Art/Anurag-singh-thakur-Rotating Sqaure/.vscode/settings.json:1]: # Folder to delete `.vscode`
- [ ] Please remove your vscode configuration folder with its content

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @Anurag-singh-thakur and thanks for your contributions.
Before we could merge your PR ;
- [ ] Please address the given feedbacks
- [ ] Please address the followings: 
  - Both contrubution folders are missing their own `meta.json` file
  Please refer to the README.md file for guidances

File: Art/Anurag-singh-thakur-Bouncing-Ball-Animation/styles.css
Comment (LaurelineP) [inline Art/Anurag-singh-thakur-Bouncing-Ball-Animation/styles.css:1]: # Incorrect folder name
- [ ] Please adjust your folder name so it does not have any space character ( which is currently the case )

File: Art/Anurag-singh-thakur-Rotating Sqaure/index.html
Comment (LaurelineP) [inline Art/Anurag-singh-thakur-Rotating Sqaure/index.html:1]: # Incorrect filename
- [ ] Please remove any space characters in your folder name ( which currently has one ) 
- [ ] Also there is a typo you could fix: `Sqaure` --> `Square`

**Comment by Anurag-singh-thakur** [comment]
It's done..please check and merge if there are any other errors then let me know .

**Comment by LaurelineP** [comment]
> It's done..please check and merge if there are any other errors then let me know .

Aloha @Anurag-singh-thakur : I am not sure I understand what exactly has been done.
- there is stil the hidden .vscod configuration that needs to be removed
- there are still incorrect css files names for both contribution
- hence there are still issues with the HTNL link tag href for both of them

Morevover - the github did not detected any changes from your hand.

Please address the feedbacks .
If this PR is stalling without changes, or maybe because it start to be hard to understand what is asked or what to read in the PR: this will be closed and then, if you still want,  you could add contribution as recommended 1 folder at a time if this could facilitate things for you.

**Comment by LaurelineP** [comment]
PR closed - feedback not addressed and stalling

**Pull request closed without merging**